docs/_build/html/_sources/tutorial.md.txt

```diff
@@ -0,0 +1,71 @@
+# Tutorial: First 15 Minutes
+This tutorial walks you through setting up Tilt for your project. It should take 15 minutes, and assumes you've already [installed Tilt](install.html). Before you begin, you may want to join the `#tilt` channel in [Kubernetes Slack](http://slack.k8s.io) for technical and moral support.
+
+Start by `cd`'ing into a project you can already build and deploy.
+
+## Example Tiltfile
+At the end of this guide, your Tiltfile will look something like this:
+```python
+# Deploy: tell Tilt what yaml to deploy
```

Comment:
[structure] hmm, the logical order of events is "build", THEN "deploy", no? maybe presenting them in that order would make it easier for users to wrap their heads around stuff? on the other hand, the order presented here is the one that works for interactive tiltifle-ing... hmm. (Idk, just musing here.)

Reply:
Right now Tilt can't build something it doesn't deploy, but it can deploy things it can't build (and should offer you a good experience as well), so I think this is the right order for onboarding, even if people tend to think of it happening the other way.